Makefile

```diff
@@ -19,7 +19,7 @@ BUILD_OUTPUT_DIRECTORY = $(BUILD_DIRECTORY)/out
 # ---------------------------------------------------------------------
 
 ELECTRON_VERSION = $(shell jq -r '.devDependencies["electron"]' package.json)
-NODE_VERSION = 6.10.3
+NODE_VERSION = 6.1.0
```

Comment:
I tried upgrading, but not even the latest `node-pre-gyp` version supports 6.10.3. 6.1 has the same ABI as 6.10.3, so everything works.